flink-examples/flink-examples-batch/src/main/scala/org/apache/flink/examples/scala/clustering/KMeans.scala

```diff
@@ -104,36 +105,52 @@ object KMeans {
 
   }
 
-  private def parseParameters(programArguments: Array[String]): Boolean = {
-    if (programArguments.length > 0) {
+  private val POINTS_PATH_OPTION: Option =
+    new Option("points").alt("P").help("The path to the input points")
+  private val CENTERS_PATH_OPTION: Option =
+    new Option("centroids").alt("C").help("The path to the input centroids")
+  private val OUTPUT_PATH_OPTION: Option =
+    new Option("output").alt("O").help("The path where the output will be written")
+  private val NUM_ITERATIONS_OPTION: Option =
+    new Option("iterations").alt("I").help("The number of iteration performed by the K-Means algorithm")
+
+  @throws(classOf[RequiredParametersException])
```

Comment:
This statement doesn't need also. We catch `RequiredParametersException` in `catch` statement.